testnet/etcd/install_etcd.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+
+ETCD_VER=v3.4.15
+ARCH=linux-amd64
+DOWNLOAD_URL="https://github.com/etcd-io/etcd/releases/download/${ETCD_VER}/etcd-${ETCD_VER}-${ARCH}.tar.gz"
+ARCHIVE="etcd-${ETCD_VER}-${ARCH}.tar.gz"
+
+install_packages curl ca-certificates
+
+# Download and extract etcdctl
+curl -s -L $DOWNLOAD_URL -o $ARCHIVE
+tar xzf $ARCHIVE etcd-${ETCD_VER}-${ARCH}/etcdctl --strip-components=1
```

Comment:
This should be dine in the `Dockerfile`.

Reply:
Yup, this needs to go, but ideally it wouldn't endup in the node Dockerfile. I was thinking about pulling the binary for the right arch in the etcd container that's used in docker compose, but it's still messy. 
I was trying to avoid wrapping `docker compose up` into some script, that builds node images, then uses those images to add etcd dependency layer, but it might be inevitable.